wardrobe/timestamp/src/lib.rs

```diff
@@ -321,6 +312,19 @@ impl<V: Verifier + From<UpForGrabs>, T: TimestampConfig + 'static> TuxedoInheren
                 .expect("Should be able to push some error");
         }
     }
+
+    #[cfg(feature = "std")]
+    fn genesis_utxos() -> Vec<Output<V>> {
+        let time = std::time::SystemTime::now()
+            .duration_since(std::time::SystemTime::UNIX_EPOCH)
+            .expect("Time is always after UNIX_EPOCH; qed")
+            .as_millis() as u64;
```

Comment:
I've been wondering why this was all feature-gated to `std`. Now I see it is because you are directly querying the time. This design is not sustainable because the native runtime is almost removed from Substrate. The way we typically pass the "local time" into the runtime is through inherent data. It comes into both `inherent_extrinsics` and `check_inherents` this way. I recommend we do the same thing here. (Or for a quick0n0dirty prototype, just take the local time as a parameter directly.)